docs/howto/operator.rst

```diff
@@ -730,6 +813,9 @@ Some arguments in the example DAG are taken from environment variables.
 Using the operator
 """"""""""""""""""
 
+You can create the operator with or without project id. If project id is missing
+it will be retrieved from the service account used. Both variants are shown:
```

Comment:
Connections???

Reply:
Yeah. That was a mental shortcut. In our environment we are using a service account to run GCP operation and we set project id from that service account as project_id parameter of the connection - so for use it's the service account's default project_id. But indeed in airflow terms it is a connection's project_id attribute value. Fixed